mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -33,17 +33,14 @@ pipeline {
                         -DMLIR_MIOPEN_DRIVER_ENABLED=1 \
                         -DMLIR_ENABLE_SQLITE=1
 
-                    # build LLVM / MLR and run tests
+                    # build LLVM / MLIR and run tests
                     cmake --build . --target check-mlir
-
-                    # Premerge checks must be invoked from the root directory
-                    cd ..
                 '''
             }
         }
         stage('Static Test') {
             steps {
-                sh """
+                sh '''
```

Comment:
I think double quotes are needed to replace $PATH with the real path. Did you test the change?

Reply:
CI has passed for this jenkinsfile. Both single/double quotes should be equivalent.